pennylane/devices/tests/test_templates.py

```diff
@@ -455,38 +455,34 @@ def circuit(params):
 
         _ = circuit(params)
 
-    def test_LocalHilbertSchmidt(self, device, tol):
+    @pytest.mark.parametrize(
+        "U", [[qml.CZ(wires=(0, 1))], (qml.CZ(wires=(0, 1)),), qml.CZ(wires=(0, 1))]
+    )
```

Comment:
Given this is a device test and not a template test, testing only on format for `U` should be fine.